app/views/users/two_factor_setup/sms_setup/new.html.haml

```diff
@@ -9,7 +9,7 @@
     Keep your account secure by using both your password and mobile phone
```

Comment:
I think this content from when sms was the only 2 factor should change now, it reads a bit funny next to the authenticator setup page. But could also wait for proper content advice.